omc3/model/accelerators/skekb.py

```diff
@@ -75,6 +75,11 @@ class SKekB(Accelerator):
     """KEK's SuperKEKB accelerator."""
     NAME = "skekb"
     RINGS = ("ler", "her")
+    RE_DICT: dict[str, str] = {
+        AccElementTypes.BPMS: r"^M*",
+        AccElementTypes.MAGNETS: r".*",  # has a variety of names...
+        AccElementTypes.ARC_BPMS: r"^M*",
+    } # naming convention for KEK elements
```

Comment:
Nice. Seems like `AccElementTypes` is not imported though.

Reply:
@wvangoet This is the more important issue to fix, imho. Not sure how this can even work for you